templates/conventions/bailleur.html

```diff
@@ -30,14 +30,20 @@ <h4 class="fr-mb-0">Entreprise bailleur</h4>
                         </strong>
                     </div>
                     <div class="fr-col-12 fr-col-lg-6">
-                        <div>{{bailleur.adresse}}</div>
+                        {% if bailleur.adresse %}
+                            <div>{{bailleur.adresse}}</div>
+                        {% endif %}
                         <div>
-                            {{bailleur.code_postal}} {{bailleur.ville}}
+                            {% if bailleur.code_postal %}{{bailleur.code_postal}}{% endif %}&nbsp;
+                            {% if bailleur.ville %}{{bailleur.ville}}{% endif %}
                         </div>
                     </div>
                     <div class="fr-col-12 fr-col-lg-6">
-                        <div class="fr-col-12">SIRET : {{bailleur.siret}}</div>
-                        <div class="fr-col-12">Capital social : {{bailleur.capital_social}}</div>
+
+                        <div class="fr-col-12">{% if bailleur.nature_bailleur == NATURE_BAILLEUR.PRIVES %}Email{% else %}SIRET{% endif %} : {{bailleur.siret}}</div>
+                        {% if bailleur.capital_social is not None %}
```

Comment:
Suggestion : Pourquoi ici on n'évalue pas directement `bailleur.capital_social` ? Il peut être nul (ou égal à zéro) ?
```suggestion
                        {% if bailleur.capital_social %}
```